health_cards.gemspec

```diff
@@ -5,16 +5,30 @@ require 'rake'
 Gem::Specification.new do |spec|
```

Comment:
Rubocop is saying this block has too many lines. This seems like it would be a place to disable it.

Reply:
I disabled Rubocop for the entire gem spec file